subworkflows/local/align_sentieon.nf

```diff
@@ -0,0 +1,69 @@
+//
+// A subworkflow to annotate structural variants.
```

Comment:
// A subworkflow of sentieon alignment and relevant post-processing steps ?

Reply:
ah yeah! :D I will fix it in a different PR :)